.gitignore

```diff
@@ -1,2 +1,3 @@
+vscode
```

Comment:
- [ ] [OPTIONAL] Hi, I believe the folder that you are trying to add to gitignore is actually called ".vscode" (notice the dot)

Reply:
Good observation. I'll solve it.